packages/git/src/browser/history/git-history-widget.ts

```diff
@@ -107,10 +107,12 @@ export class GitHistoryWidget extends GitNavigableListWidget<GitHistoryListNode>
                     const commits: GitCommitNode[] = [];
                     for (const commit of changes) {
                         const fileChangeNodes: GitFileChangeNode[] = [];
-                        const avatarUrl = await this.avartarService.getAvatar(commit.author.email);
+                        const avatarUrl = await this.avatarService.getAvatar(commit.author.email);
+                        const authorDate = new Date(commit.author.timestamp);
+                        authorDate.setUTCSeconds(commit.author.timestamp);
```

Comment:
I have reviewed the code and my recommendation was incorrect. We should do:
```typescript
const authorDate = new Date(commit.author.timestamp * 1000);
```

instead of:
```typescript
const authorDate = new Date(commit.author.timestamp);
authorDate.setUTCSeconds(commit.author.timestamp);
```

Could you please make an update, @AlexErling?